docs/guide/net.rst

```diff
@@ -70,19 +71,115 @@ This will return a list-like object which contains one or more filepaths to the
+
 Once we have loaded the dataset, if we wish to transfer all the FITS files a single call to `~dkist.io.FileManager.download` will initiate the transfer:
 
 .. code-block:: python
 
    ds.files.download()
 
-If this is the first time you have run this method a login page should open in your webbrowser to authenticate with the Globus system.
-Without overriding any of the default arguments calling this method **expects you to already have a running Globus personal endpoint** local to the Python session.
-If you wish to transfer the dataset to a non-local endpoint, you can do so by setting the ``destination_endpoint=`` keyword argument.
+If this is the first time you have run this method, or your authentication has expired, a login page should open in your webbrowser to authenticate with the Globus system.
+By default this call will download all the FITS files to the current value of ``ds.files.basepath``, i.e. next to the loaded ASDF file.
+You can override this behaviour by using the ``path=`` keyword argument.
+
+The real power of using ``download()`` however, is that you don't have to transfer the FITS files for the frames you do not wish to study (yet).
+For instance, imagine the situation where you wish to first inspect the Stokes I profile to asses the viability of the data for your analysis, using this download method you can do this and your transfer will take about a quarter of the time.
+The `~dkist.Dataset` class allows you to do this by slicing it, more details of how to do this are described in :ref:`dataset-slicing`.
+
+Continuing our example of only wanting to download the Stokes I profile we can do this by slicing the 0th element of the first array dimension (the stokes axis):
+
+.. code-block:: python
+
+   ds_I = ds[0]
+
+Then we call download on this new smaller cube:
+
+.. code-block:: python
+
+   ds_I.download()
+
+This will then only transfer the Stokes I frames.
+
+
+Downloading Complete Datasets
+#############################
+
+The alternative way of orchestrating transfers with Globus provided by the `dkist` package is the `dkist.net.transfer_whole_dataset` function.
+This will transfer the whole dataset based on a ``Fido`` search result, or the dataset ID.
+
+Given our ``Fido`` search result from earlier:
+
+.. code-block:: python
+
+   from sunpy.net import Fido, attrs as a
+   import dkist
+   import dkist.net
+
+   results = Fido.search(a.Instrument("VBI"), a.Time("2022-01-01", "2022-01-02"))
+   results
+
+If we wanted to transfer all of the datasets returned by this search we could pass the results object to `~.transfer_whole_dataset`:
+
+.. code-block:: python
+
+    transfer_whole_dataset(results["dkist"])
+
+Note how we have to extract the ``"dkist"`` table from the `~sunpy.net.fido_factory.UnifiedResponse` object, as `.transfer_whole_dataset` only operates on DKIST datasets.
+This will iterate over each dataset in the results and transfer them one-by-one showing a progress bar for each one.
```

Comment:
I'm confused by the pluralization here. The section title and function name of `transfer_whole_dataset` implies that only a single dataset will be downloaded, but here you say that it actually iterates over multiple datasets. What gives?

Reply:
Looks like the support for multiple datasets as an argument type to `transfer_whole_dataset` came later.  I think the outcome is that this block of text is fine but the name of `transfer_whole_dataset` and it's doc string need updating.

Reply:
I have renamed the function and fixed up the docstring.